app/src/main/res/layout/activity_power_source.xml

```diff
@@ -58,6 +58,19 @@
                             android:textSize="@dimen/power_card_header_text"
                             android:textStyle="bold" />
 
+                        <TextView
+                            android:id="@+id/power_card_pv1_display"
+                            android:layout_width="match_parent"
+                            android:layout_height="0dp"
+                            android:layout_margin="@dimen/power_card_large_margin"
+                            android:layout_weight="1"
+                            android:background="@drawable/control_edittext"
+                            android:fontFamily="monospace"
```

Comment:
remove font family, same for other sections of the xml file. I didn't find any font-family in the other xml files